app/helpers/hiring_staff/schools_helper.rb

```diff
@@ -0,0 +1,23 @@
+module HiringStaff::SchoolsHelper
+  def table_header_sort_by(title, column:, sort:)
+    if column == sort.column
+      order = sort.reverse_order
+      active_class = ' active'
+    else
+      order = sort.order
+    end
+
+    link_to title,
+            jobs_with_type_school_path(school_vacancy_params(sort_column: column, sort_order: order)),
+            class: "govuk-link sortby--#{order}#{active_class || ''}",
+            'aria-label': "Sort jobs by #{title} in #{order}ending order"
```

Comment:
Nice - good to see some aria attributes there 💯

Reply:
I love the use of the `order` var to get 'descending' or 'ascending', but this might be better put in a translation file - gimme a shout if you need some guidance there 👍